server/routes/goalsRouter.ts

```diff
@@ -0,0 +1,30 @@
+import { RequestHandler, Router } from 'express'
+import { Services } from '../services'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import getPrisonerData from '../middleware/getPrisonerDataMiddleware'
+import checkPrisonerInCaseload from '../middleware/checkPrisonerInCaseloadMiddleware'
+import GoalsController from '../controllers/goalsController'
+import auditPageAccessAttempt from '../middleware/auditPageAccessAttempt'
+import { Page } from '../services/auditService'
+
+export default function goalsRouter(services: Services): Router {
+  const router = Router()
+
+  const get = (path: string | string[], ...handlers: RequestHandler[]) =>
+    router.get(
+      path,
+      handlers.map(handler => asyncMiddleware(handler)),
+    )
+
+  const goalsController = new GoalsController(services.workAndSkillsPageService, services.auditService)
+
+  get(
+    '/prisoner/:prisonerNumber/goals',
```

Comment:
Similarly to my comment re the page names, would `/prisoner/:prisonerNumber/vc2-goals` be better?

Reply:
As discussed on Slack we agreed to rename this